builder/builder-git.c

```diff
@@ -51,18 +51,35 @@ git (GFile   *dir,
 
 static GFile *
 git_get_mirror_dir (const char     *url_or_path,
+                    gboolean       *is_local,
```

Comment:
Thanks for the suggestion. It already felt not 100% "right" while adding all that code for passing is_local and then not using it. Done.